web/src/routes/results/[eid]/+page.svelte

```diff
@@ -0,0 +1,259 @@
+						<tr>
+							<td>{result.name}</td>
+							<td>{result.voteCount}</td>
+							<td>
+								{#if result.isRejected}
+									Rejected
+								{:else if result.isElected}
+									Elected
+								{:else}
+									Proceed
+								{/if}
+							</td>
+						</tr>
+					{/each}
+				</tbody>
+			</table>
+			{#if results.some((e) => e.isRejected)}
+				<p>
+					<b>
+						{results
+							.filter((e) => e.isRejected)
+							.map((e) => e.name)
+							.join(", ")}
+					</b> was eliminated
+				</p>
+			{/if}
+		{/each}
+		<p>
+			After {electionOutcome.rounds} rounds of voting, the winning candidate for the post of
+			<b>{election.roleName}</b>
+			is <b>{winner}</b>
+		</p>
+		<h2>About these results</h2>
+		<p>
+			This election was held using the <b
+				><a href="https://en.wikipedia.org/wiki/Instant-runoff_voting"
+					>instant-runoff voting system</a
+				></b
+			>.
+		</p>
+		<p>
+			With instant-runoff voting, voters may rank candidates based on preference from first to last,
+			the candidate who has a majority of first-preference votes is elected. If there is no clear
+			majority, then the candidate(s) with the fewest number of votes is eliminated, and their votes
+			are re-assigned according to their next preference. This process is continued until a
+			candidate has a majority.
+		</p>
+		<p>
+			Voters may also choose to vote for <b>Re-open nominations (RON)</b>, which is a mechanism for
+			voters to not to elect any candidate to the post. If RON wins, then the post remains unfilled
+			and the election process starts from the beginning.
+		</p>
+		<p>
+			If an election is tied, then the outcome is decided by random chance, in accordance with Guild
+			of Students policy.
+		</p>
+	</div>
+</Panel>
+
+{#if $user.isAdmin}
+	<Panel title="Admin stuff" headerIcon="admin_panel_settings">
+		<div class="admin-controls">
+			<div class="control-group">
+				{#if electionOutcome.isPublished}
+					<Button
+						text="Unpublish results"
+						on:click={() => publishElectionResults(election.id, false)}
+					/>
+				{:else}
+					<Button
+						text="Publish results"
+						on:click={() => publishElectionResults(election.id, true)}
+					/>
+				{/if}
+				<Button
+					text="Delete election"
+					kind="danger"
+					on:click={() => deleteElectionDialog.showModal()}
+				/>
+			</div>
+		</div>
+	</Panel>
+	<Dialog
+		bind:dialog={deleteElectionDialog}
+		title="Delete election?"
+		on:submit={() => deleteElection(election.id)}
+	>
+		<p>This will delete this election and the associated results. This action cannot be undone.</p>
+		<svelte:fragment slot="actions">
+			<Button text="Cancel" />
+			<Button text="Delete election" kind="danger" name="submit" />
+		</svelte:fragment>
+	</Dialog>
+{/if}
+
+<style>
+	div.unpublished-callout {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		gap: 16px;
+	}
+
+	div.unpublished-callout > .button {
+		flex-shrink: 0;
+	}

```

Comment:
Probably worth switching to a grid layout seeing as it's consistently used across the project. Should be a simple change that also eliminates the need for the `.button` class.

```
	div.unpublished-callout {
		display: grid;
		grid-template-columns: 1fr auto;
		align-items: center;
		gap: 16px;
	}
```